base/applications/notepad/dialog.c

```diff
@@ -877,8 +877,8 @@ VOID DIALOG_StatusBarUpdateCaretPos(VOID)
     DWORD dwStart, dwSize;
 
     SendMessage(Globals.hEdit, EM_GETSEL, (WPARAM)&dwStart, (LPARAM)&dwSize);
-    line = SendMessage(Globals.hEdit, EM_LINEFROMCHAR, (WPARAM)dwStart, 0);
-    col = dwStart - SendMessage(Globals.hEdit, EM_LINEINDEX, (WPARAM)line, 0);
+    line = (int)SendMessage(Globals.hEdit, EM_LINEFROMCHAR, (WPARAM)dwStart, 0);
+    col = (int)(dwStart - SendMessage(Globals.hEdit, EM_LINEINDEX, (WPARAM)line, 0));
```

Comment:
Does
```suggestion
    col = dwStart - (int)SendMessage(Globals.hEdit, EM_LINEINDEX, (WPARAM)line, 0);
```
fix the error as well?
Note also that the EM_LINEINDEX could return -1 as an error value...
(and those other SendMessage calls error in case there is a problem, etc.)

Reply:
Fixed.